### ISSUE TITLE
Salesforce.identity fails when iOS time setting is a specific condition.

### ISSUE THREAD

**Issue by hmuronaka** [mike4aday/SwiftlySalesforce#70]
Salesforce.identity() fails when iOS Setting Time 24-Hour Time is off ( iOS displays am/pm time when the mode is off) . 

This error doesn't occur iOS simulators and when iOS language is English.
iOS language was Japanese when the error occcured.

The URL below is a related document.
https://developer.apple.com/library/content/qa/qa1480/_index.html

```swift
struct Identity {
  public init(from decoder: Decoder) throws {
    ....
    // failed.
    self.lastModifiedDate = try container.decode(Date.self, forKey: .lastModifiedDate) 
  } 
}
```

**Comment by hmuronaka** [comment]
Thank you for your reply.

This is the error message.

```
dataCorrupted(Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: 
"last_modified_date", intValue: nil)], debugDescription: "Date string does not match format 
expected by formatter.", underlyingError: nil))
```
And I sent fix code, please refer for updating.
https://github.com/mike4aday/SwiftlySalesforce/pull/71

**Comment by abhisheksahay** [comment]
Hi ,
I am facing the same error message .

`dataCorrupted(Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "last_modified_date", intValue: nil)], debugDescription: "Date string does not match format expected by formatter.", underlyingError: nil))
`

**Comment by mike4aday** [comment]
@abhisheksahay are you using the latest version of Swiftly Salesforce? Which language (Japanese, etc.)?

**Comment by abhisheksahay** [comment]
> @abhisheksahay are you using the latest version of Swiftly Salesforce? Which language (Japanese, etc.)?

I am  using latest version and language is english .

**Comment by mike4aday** [comment]
@abhisheksahay so far, I can't reproduce this. Could you post here the JSON response as returned by Salesforce? Thanks

**Comment by hmuronaka** [comment]
Hi,
I had same problem when I log into winter19's salesforce.
It seems that date format of lastModifiedDate of Identity is changed for winter19.

**Comment by mike4aday** [comment]
Thanks @hmuronaka I'll check that ASAP.

**Comment by mike4aday** [comment]
@hmuronaka indeed... seems the date format changed in Winter '19. Thanks for letting me know. I'll update the Swiftly Salesforce code ASAP.

@abhisheksahay FYI

**Comment by abhisheksahay** [comment]
Thanks @mike4aday .

**Comment by abhisheksahay** [comment]
Hi @mike4aday ,

Please let me know When can you fix this issue?

**Comment by mike4aday** [comment]
@abhisheksahay it should be fixed with the latest release (currently 7.1.2). Let me know if you are still seeing the problem after upgrading.

**Comment by abhisheksahay** [comment]
@mike4aday Thanks, Its fixed .
